.github/workflows/go.yml

```diff
@@ -123,6 +123,62 @@ jobs:
       - name: Clean up testdata
         run: make compatibility-testdata-cleanup
 
+  e2e-rollout-tests:
+    runs-on: aliyun
```

Comment:
separate out doesn't help, the CI runs in the same queue  on `aliyun`

Reply:
won't they run in parallel? Also, it's at least good to see that the original e2e failed in this PR.

Reply:
They are not parallel in `aliyun` runner

Reply:
now there are more than one machine